.eleventy.js

```diff
@@ -22,6 +22,7 @@ const path = require("node:path");
 const fs = require("node:fs");
 const yaml = require("js-yaml");
 const { DateTime } = require("luxon");
+const { execSync } = require("node:child_process");
```

Comment:
Is there not some way built into Eleventy to get the updated date/time? This feels a bit heavy to me.